front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -69,12 +71,18 @@ const usedModelConfigs = [
 
 // Actions
 
-const ACTION_MODES = ["GENERIC", "RETRIEVAL", "DUST_APP_RUN"] as const;
+const ACTION_MODES = [
+  "GENERIC",
+  "RETRIEVAL_SEARCH",
+  "RETRIEVAL_EXHAUSTIVE",
+  "DUST_APP_RUN",
+] as const;
 type ActionMode = (typeof ACTION_MODES)[number];
 const ACTION_MODE_TO_LABEL: Record<ActionMode, string> = {
   GENERIC: "No action (Generic model)",
-  RETRIEVAL: "Search Data Sources",
-  DUST_APP_RUN: "Execute Dust App",
+  RETRIEVAL_SEARCH: "Search Data Sources",
+  RETRIEVAL_EXHAUSTIVE: "Exhaustive Data  Sources Processing",
```

Comment:
Very very nit: 2 spaces between Data and Sources 😄

Reply:
:eye: :eagle: